lms/djangoapps/edxnotes/tests.py

```diff
@@ -179,6 +188,15 @@ def test_edxnotes_blockstore_runtime(self):
         del self.problem.descriptor.runtime.modulestore
         self.assertEqual("original_get_html", self.problem.get_html())
 
+    @patch.dict("django.conf.settings.FEATURES", {'ENABLE_EDXNOTES': True})
+    def test_edxnotes_course_about_section(self):
+        """
+        Tests that get_html is not wrapped when on the AboutBlock.
+        """
+        course_section = get_course_about_section(self.request, self.course, "short_description")
+        print(course_section)
+        self.assertEqual("TODO", course_section)
```

Comment:
Can't work out how to correctly test this; these sections seem to always return the default regardless of what I try above in setUp. Also I can't reproduce the original issue; without the fix above, this should return the edx-notes-wrapper div.